doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc

```diff
@@ -13,7 +13,7 @@ Removing or changing a provider will cause errors in the infrastructure mappings
 
 ifdef::rhv[]
 [[Cloudforms_for_rhv]]
-. Install Red Hat CloudForms 4.7.3 (CFME 5.10.3) on the Manager machine. See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Installing Red Hat CloudForms on Red Hat Virtualization].
+. Install Red Hat CloudForms 4.7.0 (with CFME 5.10.3) on the Manager machine. See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Installing Red Hat CloudForms on Red Hat Virtualization].
```

Comment:
Is there any particular reason that CF 4.7.0 is paired with 5.10.3 here?

Reply:
5.10.3 is the latest version that will support migration for CF 4.7.0, which can migrate RHV 4.2.
In 5.10.4, migration is completely broken, so we had to skip that one.
5.10.6 works with CF 4.7.6, which we can't use for IMS 1.1. RHV 4.2 did not support creating conversion hosts in CF UI, so we had to use Ansible for 1.1.